Source/Scene/FrameState.js

```diff
@@ -235,6 +235,13 @@ define([
         * @default 0.0
         */
         this.imagerySplitPosition = 0.0;
+
+        /**
+         * Distances to the near and far planes of the camera frustums
+         * @type {Number[]}
+         * @default [1.0, 1000.0]
+         */
+        this.frustumSplits = [1.0, 1000.0];
```

Comment:
Are you sure about these defaults?  Perhaps set this to `[]`.  At the start of the frame, set the `length` to zero (which does not free the memory), then `push` each distance.